### DISCUSSION TITLE
HttpTransferRequest upload with ContentType

### DISCUSSION THREAD

**Discussion by henda79** [shinyorg/shiny#1225] [Q&A] [answered]
I've noticed that when using `HttpTransferRequest` and inspecting the IFormFile on the server-side the `content-type` property is empty ?

Is there something that must be done to send the correct MIME Type ?

**Comment by aritchie** [comment, marked as answer]
The HttpTransferRequest has a header options.  Set those accordingly4

**Reply by henda79** [reply]
I though this might be the case, I was hoping the content type would be filled by the server as we are using this to filter out unwanted files, but if its just the case of adding a header then it seems pointless as anyone could pass anything.
